src/cli/telemetry/telemetry.ts

```diff
@@ -159,8 +160,10 @@ type ConfigStringKeys = keyof {
   [K in keyof d.Config as Required<d.Config>[K] extends string ? K : never]: d.Config[K];
 };
 
+const OUTPUT_TARGET_KEYS_TO_KEEP = ['type', 'typesDir', 'baseUrl'];
```

Comment:
Should we keep the 'typesDir'? Isn't it possible that folks will be providing an absolute path here? (It actually may get resolved to a full path by the compiler 🤔)

Reply:
yeah total brain fart on my end haha

I think the props we want to keep are:

```ts
const OUTPUT_TARGET_KEYS_TO_KEEP = [
  'type', 
  'external',
  'autoDefineCustomElements',
  'generateTypeDeclarations',
];
```

Reply:
I think since the latter three are booleans (or, are supposed to be booleans) we can omit those as well